package.json

```diff
@@ -46,7 +47,8 @@
     "run-example": "cd example && npm start",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "react-scripts": "react-scripts"
+    "react-scripts": "react-scripts",
+    "live-edit": "concurrently \"npm run start &\" \"cd example && npm run start\""
```

Comment:
Do you need the `&` at the end of the first command here? The examples don't show it from the `concurrently` package.

Reply:
Oh I had copy pasted this command from my terminal, we probably do not need that.